e2e/ChannelApeClient.spec.ts

```diff
@@ -145,6 +145,29 @@ describe('ChannelApe Client', () => {
       });
     });
 
+    describe('And valid order', () => {
+      context('When updating order', () => {
```

Comment:
Thanks for adding an e2e test. Can you also add a unit test to cover this? Could this have been caught with a failing unit test?